KsApi/mutations/Mutations.swift

```diff
@@ -0,0 +1,19 @@
+import Foundation
+
+public struct ChangePasswordMutation: GraphMutation {
+  var input: ChangePasswordInput
+
+  public init(input: ChangePasswordInput) {
+    self.input = input
+  }
+
+  public var description: String {
+    return """
+    mutation UpdateUserPassword($input: UpdateUserAccountInput!) {\
```

Comment:
I think mutations and queries are lowercase in the schema, so `updateUserPassword` right?